pkg/kubelet/dockershim/libdocker/fake_client.go

```diff
@@ -854,3 +871,10 @@ func (f *FakeDockerPuller) GetImageRef(image string) (string, error) {
 	}
 	return image, err
 }
+
+func (f *FakeDockerClient) GetContainerStats(id string) (*dockertypes.StatsJSON, error) {
+	f.Lock()
+	defer f.Unlock()
+	f.appendCalled(calledDetail{name: "getContainerStats"})
+	return nil, nil
```

Comment:
`return nil, fmt.Errorf("not implemented")` to make it clear that this function is not properly faked. If anyone wants to use this in a test in the future, they'll need to make sure a valid output is returned.

Reply:
Fixed.